src/get.spec.js

```diff
@@ -1,4 +1,3 @@
-import axios from 'axios'
```

Comment:
Mocking `api.axios` is not needed as `axios-mock-adapter` directly hooks onto  the `axios()` called by the instance.

Reply:
I do not understand why mocking `api.axios` it is not necessary, Kitsu now has a different instance of Axios for each of his instances. If they do not mock it, I do not see how I could run the tests. Maybe I lack knowledge on the subject.

Reply:
The mocker only creates fake responses for specific routes (i.e `GET /users`). Axios itself (in this case the instance) still operates the same, so the headers are applied to the instance and sent correctly.

Thus, creating a single mock is functionally equivalent to creating a mock for each test over and over.